internal/ioutil/tree.go

```diff
@@ -0,0 +1,58 @@
+package ioutil
+
+import (
+	"fmt"
+)
+
+const (
+	treeItemPrefix     = "├── "
+	treeItemPrefixLast = "└── "
+	subTreePrefix      = "│   "
+	subTreePrefixLast  = "    "
+)
+
+type Node struct {
+	Value    string
+	Children []*Node
+}
+
+func NewTree(value string) *Node {
```

Comment:
If the package name is `tree` instead of `ioutil`, it can just be `New()`. Later, it will be used as `tree.New()`.